LICENSES/Apache-2.0.txt

```diff
@@ -7,17 +7,17 @@ AND DISTRIBUTION
 
    1. Definitions.
 
-      
+
```

Comment:
Modifying LICENSE files does not help us. We are only the downstream of those files (see the `reuse` tool). If you want to change this, you would need to change it at the upstream/source.

Reply:
I think we can add exclusions to the pre-commit configuration file, but is it worth it? This is only whitespace.

Reply:
I've excluded LICENSES/.*

Reply:
reuse.software specifies that the LICENSE file should not be changed at all. It might even be checksums (not sure about that). Any changes make it more difficult to see if we changed the license or not. Also, as I said, making changes downstream is not the solution.